src/EditorFeatures/Core/Implementation/Classification/EditorClassifier.cs

```diff
@@ -94,6 +94,12 @@ internal static class EditorClassifier
             Order(syntaxSpans);
             Order(semanticSpans);
 
+            // Additive classifications provide additional context to the 
+            // traditional classifications. We remove these since we can only 
+            // return a single classified span for any particular span of text.
+            RemoveAdditiveClassifications(syntaxSpans);
```

Comment:
that's a pity.  Why can't we return multiple classified spans for a particular span?

Reply:
I'll revisit this. I was afraid that returning multiple classified spans for the same span would break TaggedText since it only holds a single tag for each bit of text. My concern may not be warranted.

Reply:
My guess is that it will likely break something :)  However, my preference would be to fix wahtever it breaks.  

1. if nothing breaks.  Great!
2. if something breaks, and it's easy to fix.  Great!
3. if something breaks, and it's costly to fix, then i'm ok with this approach here.

I'm pretty sure it will be '2'.  After all, if it really breaks someone that side could always do the filtering themselves as a cheap way to get around it.  (And this likely means the filtering helper should be in a common location).

Reply:
A small change to AdjustSpans was needed but the tests pass at least. =)